environments/demo/apim_appgw_config.yaml

```diff
@@ -22,7 +22,8 @@ gateways:
          - rootca_certificate_name: liberata_ca
          - rootca_certificate_name: exela_ca
          - rootca_certificate_name: pcipal_ca
```

Comment:
also did this one work? if it didn't can we delete it from the code and key vault please?

Reply:
ah this is the original one so should stay - will close this comment